File: CEMEP/Projeto_Rubia_2018/Projeto_Rubia_2018/RUBIAPPLICATION/RUBIAPPLICATION/Form12.vb
Public Class FrmConsultaEd

    Private Sub EditoraBindingNavigatorSaveItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles EditoraBindingNavigatorSaveItem.Click
        Me.Validate()
        Me.EditoraBindingSource.EndEdit()
        Me.TableAdapterManager.UpdateAll(Me.BDSystem1DataSet)

    End Sub

    Private Sub FrmConsultaEd_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        'TODO: This line of code loads data into the 'BDSystem1DataSet.Editora' table. You can move, or remove it, as needed.
        Me.EditoraTableAdapter.Fill(Me.BDSystem1DataSet.Editora)

    End Sub

    Private Sub TextBox1_TextChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles TextBox1.TextChanged
        Me.EditoraTableAdapter.FillByEditoraNome(Me.BDSystem1DataSet.Editora, TextBox1.Text)

    End Sub
End Class

File: CEMEP/Industria_Metalurgica/Backup/Industria_Metalurgica/Form12.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace Industria_Metalurgica
{
    public partial class FrmNota : Form
    {
        public FrmNota()
        {
            InitializeComponent();
        }

        private void nota_fiscalBindingNavigatorSaveItem_Click(object sender, EventArgs e)
        {
            this.Validate();
            this.nota_fiscalBindingSource.EndEdit();
            this.tableAdapterManager.UpdateAll(this.industria_MetalurgicaDataSet);

        }

        private void FrmNota_Load(object sender, EventArgs e)
        {
            // TODO: This line of code loads data into the 'industria_MetalurgicaDataSet.Clientes' table. You can move, or remove it, as needed.
            this.clientesTableAdapter.Fill(this.industria_MetalurgicaDataSet.Clientes);
            // TODO: This line of code loads data into the 'industria_MetalurgicaDataSet.Produtos' table. You can move, or remove it, as needed.
            this.produtosTableAdapter.Fill(this.industria_MetalurgicaDataSet.Produtos);
            // TODO: This line of code loads data into the 'industria_MetalurgicaDataSet.Nota_fiscal' table. You can move, or remove it, as needed.
            this.nota_fiscalTableAdapter.Fill(this.industria_MetalurgicaDataSet.Nota_fiscal);

        }

        private void button2_Click(object sender, EventArgs e)
        {
            Close(); 
        }

        private void FrmNota_KeyPress(object sender, KeyPressEventArgs e)
        {
            if (e.KeyChar == (Char)13)
                SendKeys.Send("{Tab}");
        }
    }
}
